tests/conftest.py

```diff
@@ -75,6 +75,7 @@ def pytest_configure():
             'rest_framework_gis',
             'rest_framework.authtoken',
             'crispy_forms',
+            'stored_messages',
```

Comment:
Are we relying in tests on this app being present? I couldn't find it in the changed code, but that doesn't mean it's not used ;-)